ftw/jsondump/dexterity/base.py

```diff
@@ -18,7 +18,14 @@ def extract(self, name, data, config):
         storage = self.field.interface(self.context)
         value = getattr(storage, name)
         value = self.convert(value)
-        data.update({name: value})
+        data.update({self.key: value})
+
+    @property
+    def key(self):
+        return '.'.join((
+                self.field.interface.__module__,
```

Comment:
@jone  instead of self.field.interface.__name__, and self.field.interface.__module__, you can use __identifier__